packages/node_modules/overmind/src/index.ts

```diff
@@ -106,7 +106,7 @@ export class Overmind<Config extends Configuration> implements Configuration {
       options.devtools !== false &&
       typeof window !== 'undefined'
     ) {
-      if (location.hostname === 'localhost' || options.devtools) {
+      if (location.hostname === 'localhost' && options.devtools) {
```

Comment:
Ah, I see... yeah, the check is wrong, but it actually still is. Cause now with `&&` we require you to be on localhost, which do not want to restrict.

```suggestion
       if (
       	 (location.hostname === 'localhost' && options.devtools !== false) || 		
       	 options.devtools
       ) {
```

This runs devtools automatically on localhost if not explicitly set to `false`... or if you have explicitly set something in devtools :)

Reply:
Oh I see I didn't know the expected behavior.